test/screenshot/spec/mdc-textfield/classes/focused-outlined-placeholder.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
```

Comment:
Given we have other test pages that show multiple focus states at once, it seems like we could combine this into focused-baseline-placeholder.html (and just call it focused-placeholder.html)? Save ourselves one more page load and screenshot capture.

Reply:
The reason I separated these tests out is that I specifically want to test if placeholder text and label are not overlapping. Since we can only have one element in focus state per page I had to create separate test for these 2 variants.

Reply:
Given the recent update to the styles to select on the `--focused` class instead of `:focus`, I think we can actually rely on that instead of `autofocus`. That will allow us to combine these, and should also avoid flakes due to the blinking text cursor.